lib/ddg-atb.js

```diff
@@ -37,13 +37,18 @@ function redirect(e) {
 }
 
 function update(){
-    var atb     = ss.storage.atb, 
-        setATB  = ss.storage.set_atb;
+    var atb = ss.storage.atb; 
 
-    if(!atb || !setATB){
+    if(!atb){
         return;
     }
 
+    if(!ss.storage.atb_set){
+        ss.storage.atb_set = true;
+    }
```

Comment:
return, the only place they can get `set_atb` is from atb-grabber where it pulls ATB off the DOM at time of install.